website/src/logged_in/logs/Logs.js

```diff
@@ -76,6 +87,106 @@ const columns = [
             }
         }
     },
+    {
+        field: 'daystoSetup',
+        headerName: 'Number of SetupDays',
+        description: 'Number of SetupDays',
+        sortable: true,
+        width: 200,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['setup_days']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'daystoTeardown',
+        headerName: 'Number of TearDown Days',
+        description: 'Number of TearDown Days',
+        sortable: true,
+        width: 200,
```

Comment:
Lets make this 250/275 width. We want the column header to be fully visible to the user.